docs/kata-orchestrator-reference-skills/skills/kata-configure-settings/scripts/project-root.sh

```
@@ -0,0 +1,24 @@
+# Source this at the top of any script that needs project-relative paths.
+# After sourcing, CWD is the project root (directory containing .planning/).
+#
+# Detection priority:
+#   1. KATA_PROJECT_ROOT env var (explicit override)
+#   2. CWD contains .planning/
+#   3. CWD/workspace contains .planning/ (bare repo root, prefer workspace over main)
+#   4. CWD/main contains .planning/ (bare repo root, legacy fallback)
+#   5. Error with instructions
+
+if [ -n "${KATA_PROJECT_ROOT:-}" ] && [ -d "${KATA_PROJECT_ROOT}/.planning" ]; then
+  cd "$KATA_PROJECT_ROOT"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)